### ISSUE TITLE
Fix test_jelly error: "Cannot create a consistent method resolution order (MRO) for bases object, Jellyable" on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8618]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8618|
|Type|enhancement|
|Created|2016-07-18 06:43:38Z|
|Branch|https://github.com/twisted/twisted/tree/8618-rodrigc-spread|

When trying to run test_jelly.py under Python 3, this error occurs:

```
Traceback (most recent call last):
  File "/Users/crodrigues/twisted7/twisted/trial/runner.py", line 781, in loadByName
    return self.suiteFactory([self.findByName(name, recurse=recurse)])
  File "/Users/crodrigues/twisted7/twisted/trial/runner.py", line 679, in findByName
    __import__(name)
  File "/Users/crodrigues/twisted7/twisted/test/test_jelly.py", line 17, in <module>
    class TestNode(object, jelly.Jellyable):
builtins.TypeError: Cannot create a consistent method resolution
order (MRO) for bases object, Jellyable
```

<details><summary>Searchable metadata</summary>

```
trac-id__8618 8618
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8618_rodrigc_spread 8618-rodrigc-spread
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468824218645730 1468824218645730
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/347

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 4b2c6509d589adf065da772041946b1b26d4f85e
```
#!CommitTicketReference repository="" revision="4b2c6509d589adf065da772041946b1b26d4f85e"
Merge pull request #347 from twisted/8618-rodrigc-spread

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8618

Fix MRO error on Python3 for class TestNode
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
